Packs/PAN-OS/Playbooks/playbook-PAN-OS_-_Block_Destination_Service.yml

```diff
@@ -930,16 +930,108 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "29":
+    id: "29"
+    taskid: 93958d7c-611a-412f-8e50-52f0e98b3db7
+    type: condition
+    task:
+      id: 93958d7c-611a-412f-8e50-52f0e98b3db7
+      version: -1
+      name: Is device-group "shared"?
+      description: Checks if device group is "shared", because in that case we need to push to the actual device-groups that reference firewalls.
```

Comment:
```suggestion
      description: Checks if the device group is "shared", because in that case we need to push to the actual device-groups that reference firewalls.
```